src/app/dashboard-widgets/analytical-report-widget/analytical-report-widget.component.scss

```diff
@@ -1 +1,39 @@
 @import '../../../assets/stylesheets/base';
+
+@mixin transform-translate($x, $y) {
+    -webkit-transform: translate($x, $y);
+      -ms-transform: translate($x, $y);
+          transform: translate($x, $y);
+}
+
+.card-pf-body {
+    min-height: 360px;
```

Comment:
IMO we should avoid generic classes until we add them with custom class or outer wrapper so that it won't affect other designs